.github/workflows/publish-android.yml

```diff
@@ -12,7 +12,15 @@ on:
         description: Target branch
         type: string
         required: false
+      maven_publish:
+        type: boolean
+        required: false
+        default: true
   workflow_dispatch:
+    inputs:
+      maven_publish:
+        description: 'Publish to Maven Central'
+        default: true
```

Comment:
I think we don't need the `workflow_dispatch` for the publish-android and publish-jvm workflows. They care reusable workflows not intended to be called directly.